templates/parts/loop/page-content.php

```diff
@@ -10,6 +10,15 @@
 
 	<?php
 
+	/**
+	 * primer_before_content hook.
+	 *
+	 * @hooked primer_woo_shop_message - 10
+	 *
+	 * @since 1.0.0
+	 */
+	do_action( 'primer_before_content' );
```

Comment:
@EvanHerman rather than re-use this hook, should we be doing `primer_before_page_content` instead?